blog/zh/blog/2023/02/07/apisix-ingress-with-cert-mamager.md

```diff
@@ -0,0 +1,261 @@
+---
+title: "APISIX Ingress 如何使用 Cert Manager 管理证书"
+authors:
+  - name: "张晋涛"
+    title: "Author"
+    url: "https://github.com/tao12345666333"
+    image_url: "https://github.com/tao12345666333.png"
+keywords:
+- Ingress Controller
+- 云原生
+- APISIX
+- 证书管理
+- Vault
+- cert manager
+description: 本文中介绍了 Kubernetes 中证书的默认存储方式，以及这种方式存在的一些痛点。cert-manager 的出现比较好的解决了这些问题，逐步成为了 Kubernetes 生态中证书签发/管理领域中的事实标准。
+tags: [Ecosystem]
+---
+
+> 本文中介绍了 Kubernetes 中证书的默认存储方式，以及这种方式存在的一些痛点。cert-manager 的出现比较好的解决了这些问题，逐步成为了 Kubernetes 生态中证书签发/管理领域中的事实标准。
```

Comment:
```suggestion
> Cert Manager 解决了 Kubernetes 中证书的默认存储方式的痛点，并逐步成为了 Kubernetes 生态中证书签发和管理领域中的事实标准。
```

Reply:
The quote shows up properly, but the markdown syntax says always add a space between `>`and quoted texts, see 
https://www.markdownguide.org/basic-syntax/#nested-blockquotes.

Reply:
Thank you very much, I will pay attention next time.